stable_baselines/a2c/utils.py

```diff
@@ -102,14 +102,24 @@ def conv(input_tensor, scope, *, n_filters, filter_size, stride,
     :param input_tensor: (TensorFlow Tensor) The input tensor for the convolution
     :param scope: (str) The TensorFlow variable scope
     :param n_filters: (int) The number of filters
-    :param filter_size: (int) The filter size
+    :param filter_size: either (int) or a list or tuple [int, int]. The filter size for the squared matrix,
+    or the height and width of filter if the input is a list
     :param stride: (int) The stride of the convolution
     :param pad: (str) The padding type ('VALID' or 'SAME')
     :param init_scale: (int) The initialization scale
     :param data_format: (str) The data format for the convolution weights
     :param one_dim_bias: (bool) If the bias should be one dimentional or not
     :return: (TensorFlow Tensor) 2d convolutional layer
     """
+    if isinstance(filter_size, list) or isinstance(filter_size, tuple):
+        if len(filter_size) == 2:
```

Comment:
I would rather do an assert and add a meaningful error message ;)

Reply:
New code: 
`    if isinstance(filter_size, list) or isinstance(filter_size, tuple):
        assert len(filter_size) == 2, "Filter size needs to have 2 elements, both height and width!"`  
is it better?